scripts/publish_gh_release_notes.py

```diff
@@ -0,0 +1,98 @@
+"""
+Script used to publish GitHub release notes extracted from CHANGELOG.rst.
+This script is meant to be executed after a successful deployment in Travis.
+
+Uses the following environment variables:
+* TRAVIS_TAG: the name of the tag of the current commit.
+* GH_RELEASE_NOTES_TOKEN: a personal access token with 'repo' permissions.
+
+The script also requires ``pandoc`` to be previously installed in the system.
+Requires Python3.6+.
+
+Based on code from pytest.
+https://github.com/pytest-dev/pytest/blob/master/scripts/publish_gh_release_notes.py
+Copyright Holger Krekel and others, 2004-2019.
+
+Distributed under the terms of the MIT license, pytest is free and open source software.
+"""
+import os
+import re
+import sys
+from pathlib import Path
+from typing import Text
+
+# if this needs any more dependencies, they need to be installed on travis deploy stage
+import github3
+import pypandoc
+
+
+def create_github_release(slug: Text, token: Text, tag_name: Text, body: Text):
+    """Create a github release."""
+
+    github = github3.login(token=token)
+    owner, repo = slug.split("/")
+    repo = github.repository(owner, repo)
+    return repo.create_release(tag_name=tag_name, body=body)
```

Comment:
no it wont, it creates a "Release" not a tag (these are the shiny boxes with info about the release https://github.com/pytest-dev/pytest/releases). it will fail if the release is already present but in that case 🤷‍♂